bootstrap.sh

```diff
@@ -354,12 +350,16 @@ get_shell_startup_script() {
 # Install Sentry CLI so that we can track errors that happen in this bootstrap script
 # This requires xcode CLI to be installed
 install_sentry_cli() {
+  log "Installing sentry-cli"
   if ! command -v sentry-cli &>/dev/null; then
-    log "Installing sentry-cli"
     curl -sL https://sentry.io/get-cli/ | bash
-    logk
+  fi
+  if [ -z "$CI" ]; then
```

Comment:
I add this block here because I don't want the `eval` call to happen in CI.

Reply:
what do we do if this starts failing in CI?

Reply:
What part are you referring to? We don't want it to report to Sentry; do we? I could give it a different environment if we want to.

Reply:
Oh yeah I suppose we don't want it to - the CI failing should be signal enough